src/events/Ready.ts

```diff
@@ -18,5 +19,8 @@ export class Ready implements EventHandler {
 			birthdayTimer(guild.toString(), bot);
 		});
 		launchVoice(bot);
+		let now = new Date();
+		let channel = bot.client.channels.cache.get(config.error_channel) as TextChannel;
```

Comment:
Ideally we'd check to make sure the channel we get is a textchannel, and throw an error/log if it isn't, but it's whatever, just on the configurator to make sure